server/hl7/hl7.model.js

```diff
@@ -1,8 +1,24 @@
 const mongoose = require('mongoose');
 
-const HL7FileUploadSchema = new mongoose.Schema({
-  fileName: { type: String, required: true },
-  dateAdded: { type: Date, default: () => Date.now() },
+const MessageSchema = new mongoose.Schema({
+  fileId: {
```

Comment:
Where is this field supposed to be coming from?

Reply:
Wouldn't this just be a duplicate with the mongo `_id` field.

Reply:
this comes from the subdocument within the user's document for the file that this message was parsed out of. e.g. `user.files[0]._id`